htdocs/api/v0.0.3-dev/APIBase.php

```diff
@@ -127,7 +127,9 @@ function handleRequest()
      */
     function handleETag()
     {
+        session_destroy();
```

Comment:
Won't this log out any user who makes an API call? If so, I don't think this makes sense for the release since this code is mostly deprecated anyways, we're better off waiting until the conversion from htdocs/mod_rewrite to a proper module is done, and just living with the warning until then.